Makefile

```diff
@@ -125,8 +125,8 @@ vet: #EXHELP Run go vet against code.
 
 .PHONY: bingo-upgrade
 bingo-upgrade: $(BINGO) #EXHELP Upgrade tools
-	@for pkg in $$($(BINGO) list | awk '{ print $$1 }' | tail -n +3); do \
-		echo "Upgrading $$pkg to latest..."; \
+	@for pkg in $$($(BINGO) list | awk '{ print $$3 }' | tail -n +3 | sed 's/@.*//'); do \
+		echo -e "Upgrading \033[35m$$pkg\033[0m to latest..."; \
 		$(BINGO) get "$$pkg@latest"; \
 	done
```

Comment:
Worked fine in the MAC as well
![Screenshot 2025-01-14 at 11 02 14](https://github.com/user-attachments/assets/7b52b856-4c96-4bc4-947e-da8b9d4a0692)